<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eredm√©nyek</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background-color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">* {
	background-color: rgb(70, 70, 100);
	border: 2px solid #8f8f91;
	border-radius: 10px;
	color: rgb(100, 100, 200);
	font-size: 16px;
	font-weight: bold;
}</string>
    </property>
    <widget class="QLabel" name="userScore">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>300</y>
       <width>331</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	color: whitesmoke;
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>400</y>
       <width>171</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>-1</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: rgb(70, 70, 100);
	border: 2px solid #8f8f91;
	border-radius: 10px;
	color: whitesmoke;
}

*:hover {
	background-color: rgb(120, 120, 220);
	color: white;
}</string>
     </property>
     <property name="text">
      <string>Vissza</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">* {
	background-color: #3d3d7b;
}</string>
     </property>
     <widget class="QFrame" name="learnMedal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid grey;
border-radius: 0px;
background-color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="quizMedal">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid grey;
border-radius: 0px;
background-color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="emailMedal">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid grey;
border-radius: 0px;
background-color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="badge01">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid grey;
border-radius: 0px;
background-color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="badge02">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid grey;
border-radius: 0px;
background-color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="badge03">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid grey;
border-radius: 0px;
background-color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="badge04">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid grey;
border-radius: 0px;
background-color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="badge05">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>170</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid grey;
border-radius: 0px;
background-color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QFrame" name="badge06">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>170</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">* {
border: 1px solid grey;
border-radius: 0px;
background-color: white;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
